### PR TITLE
Added Update-VCFBundle

### PR DESCRIPTION
**Pull request by GaryJBlake**
Module to immediatly download a bundle from the depot

### CODE REVIEW

File: powerVCF.psm1 [hunk 1]
```diff
@@ -1496,7 +1496,11 @@ Function Get-VCFBundle {
 
 	.EXAMPLE
     PS C:\> Get-VCFBundle -id 
-    This example gets the details of a specific bundle by its id 	 	
+    This example gets the details of a specific bundle by its id 
+    
+    .EXAMPLE
+    PS C:\> Get-VCFBundle | Where {$_.description -Match "vRealize"}
+    This example lists all bundles that have vRelaize in the description field
```
Comment (giulianoberteo) [inline powerVCF.psm1]: typo vRelaize

File: powerVCF.psm1 [hunk 2]
```diff
@@ -1533,6 +1537,41 @@ Function Get-VCFBundle {
...
+<#
+    .SYNOPSIS
+    Update a Bundle for downloading from depot
+	
+    .DESCRIPTION
+    Triggers an immediate download. Only one download can triggered for a Bundle. 
+	
+    .EXAMPLE
+    PS C:\> Update-VCFBundle -id 7ef354ab-13a6-4e39-9561-10d2c4de89db
+    This example gets the list of bundles and all details
```
Comment (PowerVCF) [inline powerVCF.psm1]: Example text needs to be updated

File: powerVCF.psm1 (hunk 1 above)
Comment (GaryJBlake) [reply powerVCF.psm1]: Fixed

File: powerVCF.psm1 (hunk 2 above)
Comment (GaryJBlake) [reply powerVCF.psm1]: Fixed

**Pull request merged**